include/RAJA/policy/PolicyBase.hpp

```diff
@@ -69,16 +69,26 @@ enum class Pattern {
   taskgraph,
 };
 
+enum class Platform {
+  undefined = 0,
+  host = 1,
+  cuda = 2,
+  omp_target = 4
+};
+
 struct PolicyBase {
 };
 
 template <Policy P = Policy::undefined,
           Launch L = Launch::undefined,
-          Pattern Pat = Pattern::undefined>
+          Pattern Pat = Pattern::undefined,
+          Platform Plat = Platform::host
```

Comment:
Yeah this was motivated by laziness :) 

The comment you raise though makes having a default sensible. Otherwise we have to override on a policy by policy basis.